stripe-ui-core/src/main/java/com/stripe/android/uicore/image/LoadedImage.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.uicore.image
+
+import android.graphics.Bitmap
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class LoadedImage(
+    val contentType: ContentType,
+    val bitmap: Bitmap,
+) {
+    constructor(
+        contentType: String,
+        bitmap: Bitmap,
+    ) : this(
+        contentType = from(contentType),
+        bitmap = bitmap,
+    )
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    sealed interface ContentType {
+        val value: String
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        data object Jpeg : ContentType {
+            override val value = "image/jpeg"
+        }
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        data object Png : ContentType {
+            override val value = "image/png"
+        }
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        data object Webp : ContentType {
+            override val value = "image/webp"
+        }
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        data class Unknown(override val value: String) : ContentType
+    }
+
+    private companion object {
+        fun from(value: String) = when (value) {
+            ContentType.Jpeg.value -> ContentType.Jpeg
+            ContentType.Png.value -> ContentType.Png
+            ContentType.Webp.value -> ContentType.Webp
+            else -> ContentType.Unknown(value)
+        }
```

Comment:
Probably not super necessary, but we can clean this up and future-proof this (say we ever support GIF 🙃 ).

```suggestion
        fun from(value: String): ContentType {
            return entries.find { it.value == value } ?: ContentType.Unknown(value)
        }
```